src/components/StafferPage/index.jsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { StafferForm } from './StafferForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+class StafferPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleStafferCreate = this.handleStafferCreate.bind(this);
+    this.handleStafferEdit = this.handleStafferEdit.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchStafferOptions();
+    this.props.fetchStafferInfo();
+    this.setStartedFetching();
+  }
+
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
+  }
+
+  prepareSocialLinks(links) {
+    if (!links) {
+      return [];
+    }
+
+    return links.map(link => (
+      ({
+        ...link,
+        id: null,
+      })
+    ));
+  }
+
+  prepareAreasOfExpertise(areasOfExpertise) {
+    if (!areasOfExpertise) {
+      return [];
+    }
+
+    return areasOfExpertise.map(area => ({ id: null, value: area.value }));
+  }
+
+  preparePosition(position) {
+    return {
+      title: position.title,
+      organization: position.organization_id,
+      organization_override: null,
+    };
+  }
+
+  handleStafferCreate(fieldValues) {
+    const stafferData = {
+      ...fieldValues,
+      profile_image: fieldValues.profile_image.medium.url,
```

Comment:
Do we not want the whole object here?

Reply:
This makes the eventual discovery call correct 🤷‍♂️

Reply:
Seems like it?  I'd be happy to pair with you to confirm the behavior